js-exercises/sum-primes/sumPrimes.js

```diff
@@ -0,0 +1,24 @@
+function sumPrimes(num) {
+  let prime = [];
+
+  for (var i = 2; i <= num; i++) {
+    if (isPrime(i)) {
+      prime.push(i);
+    }
+  }
+
+  function isPrime(num) {
+    for (var j = 2; j <= (num / 2); j++) {
```

Comment:
The check is not till `num / 2`, you just have to check till square root of the number. 
```suggestion
    for (var j = 2; j <= Math.sqrt(num); j++) {
```

Reply:
Also, don't use `var`s.